Resources/Prototypes/DeltaV/Entities/Objects/Decoration/flora.yml

```diff
@@ -0,0 +1,100 @@
+- type: entity
+  id: BushBase
```

Comment:
should have Base in its ID since its intended to be mapped

Reply:
👍